src/cli/init_cmd.ml

```diff
@@ -8,22 +8,26 @@ let with_file : string -> (out_channel -> unit) -> unit = fun file fn ->
 
 let write_makefile : out_channel -> unit = fun oc ->
   Printf.fprintf oc
-    "LP_SRC = $(shell find . -type f -name \"*.lp\")\n\
-     LP_OBJ = $(LP_SRC:%%.lp=%%.lpo)\n\
+    ".POSIX:\n\
+     SRC = nat.lp main.lp\n\
```

Comment:
using shell find is not posix?

Reply:
Not it isn't, POSIX makefiles do not implement external shell calls, for instance, in BSD it's performed using `LP_SRC != find ...`